lib/_tls_wrap.js

```diff
@@ -1113,6 +1114,11 @@ Server.prototype.setSecureContext = function(options) {
   else
     this.crl = undefined;
 
+  if (options.sigalgs !== undefined && options.sigalgs !== null)
```

Comment:
sorry, I think I suggested allowing `null`, but having looked, I think allowing just string or undefined is current convention.